wordle-web/src/components/LetterBase.vue

```diff
@@ -1,15 +1,35 @@
 <template>
-  <v-btn :color="props.color">
+  <v-btn class="shadow-5, elevation-10" id="gradient" :color="props.color">
     {{ props.char }}
   </v-btn>
 </template>
 
+<style scoped>
+#gradient {
+  background: linear-gradient(to right, #e1e1e1, #747474);
```

Comment:
This is really cool! My partner and I had a hard time trying to figure out how to do a color gradient on the buttons and in fact didn't get it done in time for the initial commit. Now I am curious how one could make the buttons keep that gradient after they change color.